nucleus/parent/pom.xml

```diff
@@ -83,41 +83,41 @@
         <!-- Jakarta API Versions -->
 
         <!-- Jakarta EL -->
-        <jakarta.el-api.version>4.0.0</jakarta.el-api.version>
-        <jakarta.el.version>4.0.2</jakarta.el.version>
+        <jakarta.el-api.version>5.0.0</jakarta.el-api.version>
+        <jakarta.el.version>5.0.0-M1</jakarta.el.version>
 
         <!-- Jakarta Servlet -->
-        <servlet-api.version>5.0.0</servlet-api.version>
+        <jakarta.servlet-api.version>6.0.0</jakarta.servlet-api.version>
 
         <!-- Jakarta Validation -->
-        <jakarta.validation.version>3.0.0</jakarta.validation.version>
+        <jakarta.validation-api.version>3.0.1</jakarta.validation-api.version>
         <hibernate-validator.version>7.0.1.Final</hibernate-validator.version>
 
         <!-- Jakarta Web Services -->
         <webservices.version>3.0.3</webservices.version>
 
         <!-- Jakarta Inject -->
-        <jakarta.inject-api.version>2.0.0</jakarta.inject-api.version>
+        <jakarta.inject-api.version>2.0.1.MR</jakarta.inject-api.version>
 
         <!-- Jakarta XML Binding -->
-        <jakarta.jaxb-api.version>3.0.1</jakarta.jaxb-api.version>
-        <jakarta.jaxb-impl.version>3.0.2</jakarta.jaxb-impl.version>
+        <jakarta.jaxb-api.version>4.0.0-RC2</jakarta.jaxb-api.version>
+        <jakarta.jaxb-impl.version>4.0.0-M1</jakarta.jaxb-impl.version>
 
         <!-- Jakarta REST -->
         <jax-rs-api.spec.version>3.0</jax-rs-api.spec.version>
-        <jax-rs-api.impl.version>3.0.0</jax-rs-api.impl.version>
-        <jersey.version>3.0.3</jersey.version>
+        <jax-rs-api.impl.version>3.1.0</jax-rs-api.impl.version>
+        <jersey.version>3.1.0-SNAPSHOT</jersey.version>
```

Comment:
Jersey has 3.1.0-M1 in sonatype already

Reply:
Thanks, I'll try that one

Reply:
Unfortunately M1 is not strong enough, I get the following error again, which blocks the startup of the server completely:

```
3. java.lang.IllegalStateException: Unable to resolve
    org.glassfish.main.web.jersey-mvc-connector [31]
    missing requirement
        &(package = org.glassfish.jersey.server.mvc.jsp) (version >= 3.1.0) (!(version >= 4.0.0))
        caused by:
            Unable to resolve
                org.glassfish.jersey.ext.jersey-mvc-jsp [158]
                missing requirement
                    &(package = org.glassfish.jersey.server.mvc.internal) (version >= 3.1.0) (!(version >= 4.0.0))
                    caused by:
                        Unable to resolve
                            org.glassfish.jersey.ext.jersey-mvc [276]
                            missing requirement
                                &(package = jakarta.servlet) (version >= 5.0.0) (!(version >= 6.0.0)))]]
```

Reply:
I can try to hotpatch M1 though and try to make it as good as the current 3.1 branch.

Reply:
Right, 3.1.0-M1 does not have the servlet OSGi headers set properly, yet

Reply:
M1 is probably also missing the CDI fixes. If I hot patch the archive, I get this:

```
 java.lang.NoSuchMethodError: 'jakarta.enterprise.inject.spi.InjectionTarget jakarta.enterprise.inject.spi.BeanManager.createInjectionTarget(jakarta.enterprise.inject.spi.AnnotatedType)'
	at org.glassfish.jersey.ext.cdi1x.servlet.internal.CdiExternalRequestScopeExtension.afterBeanDiscovery(CdiExternalRequestScopeExtension.java:72)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.jboss.weld.injection.StaticMethodInjectionPoint.invoke(StaticMethodInjectionPoint.java:95)
```

It's probably better for GF to keep using the branch for now, and wait till there's a M2 with the necessary fixes.

Reply:
-m2 now available, and added. Thanks!